src/dotnet/install.sh

```diff
@@ -6,7 +6,7 @@
 #
 # Docs: https://github.com/devcontainers/features/tree/main/src/dotnet
 # Maintainer: The Dev Container spec maintainers
-DOTNET_VERSION="${VERSION:-"latest"}"
+DOTNET_VERSION="${VERSION:-"lts"}"
```

Comment:
I'm not sure if this was just for testing but otherwise I wouldn't change the default version